lua/telescope/_extensions/file_browser/utils.lua

```diff
@@ -147,4 +147,46 @@ fb_utils.group_by_type = function(tbl)
   end)
 end
 
+local function add_tele_prefix(msg)
+  if not msg:match "^%[telescope%]" then
+    msg = "[telescope] " .. msg
+  end
+  return msg
+end
+
+-- Move this and above functions into telescope core?
+fb_utils.tele_notify = function(msg, log_level, opts)
```

Comment:
I dont get what you try to achieve with both of these functions. Any styling should be handled by vim.notify, thats the whole point.

For example i saw this:
```
fb_utils.tele_notify "\nRemoving files aborted!"
```

Which results in this output. If you are using https://github.com/rcarriga/nvim-notify

![image](https://user-images.githubusercontent.com/15233006/151626284-552c9a90-092e-411d-a702-a659722e5f4d.png)

the max wrapper around vim.notify i would accept in telescope core is this

```lua
utils.notify = function(msg, log_level, opts)
  vim.notify("[telescope]" .. msg, log_level, opts)
end
```

And then we could replace all the print 
```lua
print "[telescope] Nothing currently selected"
```
with 
```lua
utils.notify "Nothing currently selected"
```
in core

Reply:
Or we can use the title possibility of `nvim-notify` if its available. So if not `[telescope] msg` else `vim.notify(msg, log_level, { title = "telescope" })`

Reply:
> Or we can use the title possibility of `nvim-notify` if its available. So if not `[telescope] msg` else `vim.notify(msg, log_level, { title = "telescope" })`

😋🎉  bingo